packages/slice-machine/components/Changelog/versionDetails/index.tsx

```diff
@@ -84,20 +84,7 @@ export const VersionDetails: React.FC<VersionDetailsProps> = ({
           packageManager={packageManager}
         />
 
-        {selectedVersion.releaseNote ? (
```

Comment:
why you removed this ? This cover the fact that if the github api doesn't respond we still display partial content no ?